### ISSUE TITLE
[high] Filters display bug and UI design improvements needed

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-eval-tools#72]
Passing along this comment:
> I've checked my pages with several tools listed on that page, but I found it very difficult to navigate. Descriptions help, but I think there should be more filters. For example there should be a way to display only online or local tools (with an additional filter for supported platforms), free/paying ones, with or without [in-page feedback].

/me thinks some of those Filters are already there. In which case, the issue is UI design. Indeed, with this design, I can envision a user skimming down the left column and not recognizing the other categories of filters.
... /me checks a few things...
In Firefox, I'm not getting the twisties before the filter categories:
![image](https://user-images.githubusercontent.com/6991814/110022281-5762fe00-7cf1-11eb-9a78-ad781cdf2d99.png)

They are showing up in edge:
![image](https://user-images.githubusercontent.com/6991814/110022392-7b264400-7cf1-11eb-9a22-b36a995bff75.png)

So maybe browser inconsistency is part of the problem

**Comment by jean-gui** [comment]
Indeed, I hadn't noticed the other categories in the filter column. The first category is so long that the others are easy to miss as they are relegated to the end of the page.
I also confirm that I don't see the triangles before filter categories in Firefox, which doesn't help noticing them.

**Comment by shawna-slh** [comment]
@nitedog
I think it would be good to **address this issue asap**, since the redesign will take a while. That is:
* [ ] at a minimum, get the twisties visible in Firefox and other browsers
* [ ] consider other quick fixes to make it more apparent there are other filters, e.g.,
  * [ ] maybe have all collapsed by default

I wonder if this alone will go a long way to addressing the issues and critiques with this list? That is, if others aren't seeing the filters, no wonder they think the list is overwhelming and unusable! (fyi @sharronrush )
